dist/privacy-sandbox.js

```diff
@@ -0,0 +1,502 @@
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'leaveAdInterestGroup', false)) {
+      result['protectedAudienceAPI']['interestGroups']['leaveAdInterestGroup'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'updateAdInterestGroups', false)) {
+      result['protectedAudienceAPI']['interestGroups']['updateAdInterestGroups'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'clearOriginJoinedAdInterestGroups', false)) {
+      result['protectedAudienceAPI']['interestGroups']['clearOriginJoinedAdInterestGroups'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'runAdAuction', false)) {
+      result['protectedAudienceAPI']['runAdAuction'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'generateBid', false)) {
+      result['protectedAudienceAPI']['generateBid'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'scoreAd', false)) {
+      result['protectedAudienceAPI']['scoreAd'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'reportWin', false)) {
+      result['protectedAudienceAPI']['reportWin'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+    if (checkResponseBody(request, 'reportResult', false) || request.response_body.includes('sendReportTo', false)) {
+      result['protectedAudienceAPI']['reportResult'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+
+
+    /***************************************************************************
+     * Related Website Set
+     * Documentation: https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API
+     * Test site(s):
+     **************************************************************************/
+
+    if (checkResponseBody(request, 'document.hasStorageAccess\(\s*\)')) {
+      // [javascript] document.hasStorageAccess()
+      result['relatedWebsiteSet']['hasStorageAccess'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+
+    if (checkResponseBody(request, 'document.hasUnpartitionedCookieAccess\(\s*\)')) {
+      // [javascript] document.hasUnpartitionedCookieAccess()
+      result['relatedWebsiteSet']['hasUnpartitionedCookieAccess'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+
+    if (checkResponseBody(request, 'document.requestStorageAccess\(')) {
+      // [javascript] document.requestStorageAccess(types: optional)
+      result['relatedWebsiteSet']['requestStorageAccess'].push(requestDomain);
+      apiCallerAdd(requestDomain);
+    }
+
+    if (checkResponseBody(request, 'document.requestStorageAccessFor\(')) {
+      // [javascript] document.requestStorageAccessFor(requestedOrigin)
+      result['relatedWebsiteSet']['requestStorageAccessFor'].push(requestDomain);
```

Comment:
Is this origin or domain?

Reply:
The API call takes an origin as argument, but we are logging the domain from which the request making the call is coming from. So, `result['relatedWebsiteSet']['requestStorageAccessFor'].push(requestDomain);`

Reply:
OK makes sense.